data/wesionary-team.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "wesionaryTEAM Co. Ltd.",
+    "establishedAt":1580667300,
```

Comment:
The code format is still not correct. There should be a space between every `key` and `value` pairs. Refer other json files.